umh.docs.umh.app/content/en/docs/features/connectivity/introduction-asset-management.md

```diff
@@ -4,51 +4,60 @@ content_type: task
+### Technical Challenges Examples:
 
 - **Uncommon Protocols:** Handling rare communication protocols.
 - **Non-Ethernet Protocols:** Dealing with non-standard connections, like a 4-20 mA sensor or a USB-connected barcode reader.
 - **Monotonous Integrations:** Numerous data sources in the same format can lead to tedious, error-prone setup processes. Templating is a key solution here.
 - **High Variance in Data Sources:** For instance, different QA stations in production lines might use slightly varied REST API versions, necessitating extensive customization.
 
-## Organizational Challenges Examples:
+### Organizational Challenges Examples:
 
 - **IT and OT Tool Mismatch:** Advanced IT tools like Spark/Flink may be challenging for OT personnel who have crucial domain knowledge.
 - **OT Tools’ Limitations:** Traditional OT tools often struggle in modern IT environments, lacking features like Docker compatibility, monitoring, automated backups, or high availability.
 
-## Recommended Tools in UMH
+## What can I do with it?
+
+You can manage connectivity effectively; for example, you can create, configure, and delete connections using tools in a comfortable way. 
+
 
-To address these challenges, UMH recommends a combination of tools for effective connectivity:
+## How can I use it?
 
-### 1. Node-RED
+To address these challenges, UMH recommends a combination of the following tools for effective connectivity:
 
-- **Description:** A leading open-source tool for IIoT connectivity, widely accepted alongside Kepware.
-- **Advantages:** Vast global community, extensive connectors, Docker compatibility, user-friendly for OT personnel.
-- **Disadvantages:** Reliability may vary.
-- **UMH Recommendation:** Primarily for prototyping and integrating parts of the shopfloor that demand high levels of customization and domain knowledge.
+### Node-RED
 
-[Learn more about Node-RED's role in the industrial IoT](/docs/getstarted/noderedarticle/)
+Node-RED is a leading open-source tool for IIoT connectivity, widely accepted alongside Kepware. We recommend this tool for prototyping and integrating parts of the shop floor that demand high levels of customization and domain knowledge.
+- Advantages: Vast global community, extensive connectors, Docker compatibility, user-friendly for OT personnel.
+- Disadvantages: Reliability may vary.
 
-### 2. benthos-umh
+### benthos-umh
 
-- **Description:** UMH's custom extension of the Benthos project, enhanced with an OPC UA plugin.
-- **Advantages:** Ideal for reliably moving medium-sized data volumes, ease of deployment and management.
-- **Disadvantages:** Technical expertise required, longer development cycles.
-- **UMH Recommendation:** For scenarios involving the extraction of large data volumes in a standardized format.
+The benthos-umh is UMH's custom extension of the Benthos project, enhanced with an OPC UA plugin. It is recommended for scenarios involving the extraction of large data volumes in a standardized format.
```

Comment:
enhanced woth an OPC UA plugin. this sentence does not really make sense. benthos-umh is a fork of benthos with the most significant change being an additional OPC UA input plugin.